### DISCUSSION TITLE
How to switch the focus between left dock and editor with a shortcut?

### DISCUSSION THREAD

**Discussion by volkan0x** [zed-industries/zed#10519] [Feature Requests]
I've been wondering how to switch focus between editor and the left dock. 

https://github.com/zed-industries/zed/assets/68243739/cd9998ae-a62e-44ef-9428-0eeddfb61deb

**Comment by alygin** [comment]
You're looking for the `project panel: toggle focus` command. The default keybinding is `cmd-shift-e`.

<img width="553" alt="изображение" src="https://github.com/zed-industries/zed/assets/2101250/f4dd1767-1e70-41d9-9ca8-477fdf21ab91">

**Reply by volkan0x** [reply]
`cmd-shift-e` works but I don't have this option on my keymap.json file for changing.

**Reply by alygin** [reply]
This should work, just change the keys to your liking:

```json
{
  "context": "Workspace",
  "bindings": {
    "cmd-shift-e": "project_panel::ToggleFocus"
  }
}
```
